cvmfs/run_cloudtests.sh

```diff
@@ -25,14 +25,28 @@ has_platform_parameter 'test'  "$vm_desc" || die "VM parameter .test missing"
 has_platform_parameter 'ami'   "$vm_desc" || die "VM parameter .ami missing"
 has_platform_parameter 'user'  "$vm_desc" || die "VM parameter .user missing"
 
+# exports for scripts in mac and linux
```

Comment:
As described above, please make those *proper* environment variables with a namespace prefix and all capitals.